<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gate_editor_ui</class>
 <widget class="QDialog" name="gate_editor_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gate Editor</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>301</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Nom</string>
   </property>
   <widget class="QLineEdit" name="editName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Point 1</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Longitude Est</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lon_deg_1">
       <property name="suffix">
        <string> 째</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-189.000000000000000</double>
       </property>
       <property name="maximum">
        <double>189.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lon_min_1">
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
       <property name="value">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Latitude Nord</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lat_deg_1">
       <property name="suffix">
        <string> 째</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-89.000000000000000</double>
       </property>
       <property name="maximum">
        <double>89.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lat_min_1">
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
       <property name="value">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>301</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Point 2</string>
   </property>
   <widget class="QWidget" name="layoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Longitude Est</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lon_deg_2">
       <property name="suffix">
        <string> 째</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-189.000000000000000</double>
       </property>
       <property name="maximum">
        <double>189.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lon_min_2">
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
       <property name="value">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Latitude Nord</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lat_deg_2">
       <property name="suffix">
        <string> 째</string>
       </property>
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>-89.000000000000000</double>
       </property>
       <property name="maximum">
        <double>89.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="lat_min_2">
       <property name="suffix">
        <string> '</string>
       </property>
       <property name="maximum">
        <double>59.990000000000002</double>
       </property>
       <property name="value">
        <double>59.990000000000002</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>81</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btOk">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btCancel">
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btDelete">
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>28</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btCopy">
      <property name="text">
       <string>Copier</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btPaste">
      <property name="text">
       <string>Coller</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>301</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Couleur</string>
   </property>
   <widget class="QPushButton" name="chgColor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>75</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Changer</string>
    </property>
   </widget>
   <widget class="QFrame" name="color_previw">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btOk</tabstop>
  <tabstop>btCancel</tabstop>
  <tabstop>btDelete</tabstop>
  <tabstop>btCopy</tabstop>
  <tabstop>btPaste</tabstop>
  <tabstop>editName</tabstop>
  <tabstop>lat_deg_1</tabstop>
  <tabstop>lat_min_1</tabstop>
  <tabstop>lon_min_1</tabstop>
  <tabstop>lat_min_2</tabstop>
  <tabstop>lon_min_2</tabstop>
  <tabstop>chgColor</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btOk</sender>
   <signal>clicked()</signal>
   <receiver>gate_editor_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>363</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCancel</sender>
   <signal>clicked()</signal>
   <receiver>gate_editor_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDelete</sender>
   <signal>clicked()</signal>
   <receiver>gate_editor_ui</receiver>
   <slot>gate_remove()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>326</x>
     <y>121</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btCopy</sender>
   <signal>clicked()</signal>
   <receiver>gate_editor_ui</receiver>
   <slot>gate_copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>256</y>
    </hint>
    <hint type="destinationlabel">
     <x>323</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btPaste</sender>
   <signal>clicked()</signal>
   <receiver>gate_editor_ui</receiver>
   <slot>gate_paste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>321</x>
     <y>296</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chgColor</sender>
   <signal>clicked()</signal>
   <receiver>gate_editor_ui</receiver>
   <slot>gate_chgColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>139</x>
     <y>283</y>
    </hint>
    <hint type="destinationlabel">
     <x>218</x>
     <y>244</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>gate_remove()</slot>
  <slot>gate_copy()</slot>
  <slot>gate_paste()</slot>
  <slot>gate_chgColor()</slot>
 </slots>
</ui>
